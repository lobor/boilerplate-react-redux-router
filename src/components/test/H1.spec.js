import React from 'react'
import styled from 'styled-components'
import renderer from 'react-test-renderer'
import 'jest-styled-components'

import Column from './../column'

test('render column', () => {
  const tree = renderer.create(<Column />).toJSON()
  expect(tree).toMatchSnapshot()
})
