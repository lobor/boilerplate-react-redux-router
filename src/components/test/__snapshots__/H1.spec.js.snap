// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render column 1`] = `
.c0 {
  font-size: 1rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  margin-right: 10px;
}

.c0:last-child {
  margin: 0;
}

@media (max-width:25em) {
  .c0 {
    margin: 0;
  }
}

<div
  className="c0"
/>
`;
